### DISCUSSION TITLE
What are the limitations of Functions?

### DISCUSSION THREAD

**Discussion by es-ki** [gatsbyjs/gatsby#34859] [Help] [answered]
Hello.  
I would like to thank the Gatsby team.

I would like to know the various limitations, such as the maximum time to execute Functions and the maximum size of the response body.
The following page shows some of the limitations regarding Functions.  
[Pricing | Gatsby](https://www.gatsbyjs.com/pricing/)  

[Limits – Vercel Docs](https://vercel.com/docs/concepts/limits/overview#serverless-function-execution-timeout)  
Alternatively, I would like to know if there is a page with detailed limitations, like the Vercel document above.

**Comment by LekoArts** [comment, marked as answer]
Hi! You can reach the Cloud support here: https://www.gatsbyjs.com/support/
They'll be able to answer this. Thanks!
